app/src/main/java/org/coepi/android/ui/extensions/ProgressBarViewExtension.kt

```diff
@@ -0,0 +1,24 @@
+package org.coepi.android.ui.extensions
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import org.coepi.android.R
+
+
+class ProgressBarViewExtension : LinearLayout {
```

Comment:
What's the purpose of this custom view? I see that it basically only sets `isClickable` to `true`, which can also be done in XML?

Reply:
Making these changes here allows us to keep the minimal definition of the views from the XML. Using this, I only had to change a couple of lines (which I believe were addressed in another comment), so in the future when adding more progress bars, we should be able to include the custom view with minimal XML statements.

Reply:
I meant compared to making it reusable at XML level https://developer.android.com/training/improving-layouts/reusing-layouts, since we don't seem to need anything programmatic.